packages/components/src/components/common/inputbadge/inputBadge.tsx

```diff
@@ -10,6 +10,8 @@ import { splitNewLine } from '@/utils/string';
 export type InputBadgeProps = {
   value: string;
   badgeProps: Array<BadgeProps>;
+  style?: CSSProperties;
+  className?: string;
```

Comment:
Let me be clear this is not wrong.
But mostly all other components just reuse the props from the root element and just pass them through.
Why not just do this here too?